x-pack/plugins/infra/public/containers/waffle/nodes_to_wafflemap.ts

```diff
@@ -51,7 +52,7 @@ function findOrCreateGroupWithNodes(
         ? i18n.translate('xpack.infra.nodesToWaffleMap.groupsWithNodes.allName', {
             defaultMessage: 'All',
           })
-        : last(path).label,
+        : (lastPath && lastPath.label) || 'No Group',
```

Comment:
Just out of curiosity: Have the changes to this file slipped in by accident or do they have a relation to the flyout that I don't see? :wink:

Reply:
Something changed in our env, either the linter or new lodash definitions, that has caused TSLint to start complaining. I know it didn't complain originally when we wrote this the first time.

Reply:
Actually the complaint from the compiler is that `last(path)` could possibly be `undefined` which I guess is technically true since we can't tell the compiler that there must be at least one element for the `InfraNodePath`